platform/features/plot/src/PlotController.js

```diff
@@ -165,6 +184,19 @@ define(
                 }
             }
 
+            // Respond to a display bounds change (requery for data)
+            function changeDisplayBounds(event, bounds) {
+                self.pending = true;
+                releaseSubscription();
+                throttledRequery();
+                setBasePanZoom(bounds);
+            }
+
+            // Reestablish/reissue request for telemetry
+            throttledRequery = throttle(function () {
```

Comment:
Handling this throttling in one place (on the representer, before throwing the event) would result in not having to implement it in each view.  I'd even push for the time controller not to broadcast the event until the end of the user interaction.

Side note: How do you plan to handle request cancellation?  Server requests will queue up and will be served in the order they were queued; canceling those requests when they are no longer needed is key to ensuring performant view updates.